contracts/src/contracts/utils.py

```diff
@@ -534,10 +542,14 @@ def closeAuction(client: AlgodClient, appID: int, closer: Account):
             auction before it starts. Otherwise, this can be any account.
     """
     appGlobalState = getAppGlobalState(client, appID)
+    appCreator = getAppCreator(client, appID)
 
     nftID = appGlobalState[b"nft_id"]
 
-    accounts: List[str] = [encoding.encode_address(appGlobalState[b"seller"])]
+    accounts: List[str] = [
+        encoding.encode_address(appGlobalState[b"seller"]),
+        appCreator,
+    ]
```

Comment:
Application creator address is required to be able to send the royalty fee to that account.